vignettes/identifying_edits.Rmd

```diff
@@ -0,0 +1,212 @@
+---
+title: "Finding Candidates for Edits"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{Finding Candidates for Edits}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r, include = FALSE}
+# can run manually on datascience by removing the `eval = FALSE` flag.
+wiktionary_dump <- "/shared/wiktionary/all_wiktionary_en.rds"
```

Comment:
Oh, actually... I'd rather not reference the directory structure in this, since it's a public package. Probably add a note about that being set up in your environment?